aws/resource_aws_eip_association_test.go

```diff
@@ -437,7 +421,35 @@ resource "aws_eip_association" "test" {
 
 func testAccAWSEIPAssociationConfig_spotInstance(rInt int) string {
 	return composeConfig(
-		testAccAWSSpotInstanceRequestConfig(rInt), `
+		testAccLatestAmazonLinuxHvmEbsAmiConfig(),
+		testAccAvailableEc2InstanceTypeForRegion("t3.micro", "t2.micro"),
```

Comment:
Since we are locking to an Availability Zone anyways:

```suggestion
		testAccAvailableEc2InstanceTypeForAvailabilityZone("aws_subnet.test.availability_zone", "t3.micro", "t2.micro"),
```